charts/postgres-operator/values.yaml

```diff
@@ -0,0 +1,92 @@
+image:
+  registry: registry.opensource.zalan.do
+  repository: acid/smoke-tested-postgres-operator
```

Comment:
It's a testing image, a proper image is mentioned on the release page: registry.opensource.zalan.do/acid/postgres-operator:v1.1.0

Reply:
fixed